bedrock/mozorg/templates/mozorg/home/includes/featured-vpn.html

```diff
@@ -4,6 +4,11 @@
   file, You can obtain one at https://mozilla.org/MPL/2.0/.
 #}
 
+
+{% block page_css %}
+    {{ css_bundle('home-featured-vpn') }}
+{% endblock %}
+
```

Comment:
I moved this promo's CSS bundle into its HTML file, because it was interfering with banner conditionals when it was placed in the `home-new.html` file.

Reply:
Typically we try and stick to putting CSS in the head. I don't think this is causing any problems here in newer browsers though. However the `page_css` block is meant for page level CSS, so using that block name here should probably be avoided.

I took a look, but I can't see any conflicting variables of selectors in `home-new.html`?

Reply:
Ah, seems like I was adding the VPN promo conditional switch for the CSS bundle within the same conditional block as the banners and what was causing a clash. I separated them and both seem to function well now.